research/delf/delf/python/training/global_features_utils.py

```diff
@@ -1,229 +0,0 @@
-# Copyright 2021 The TensorFlow Authors All Rights Reserved.
```

Comment:
I see you're deleting this file. Just leaving this comment here to double-check that this was intentional. If it was intentional, please ignore my comment :)

Reply:
This module is already in a different PR ([PR_9902](https://github.com/tensorflow/models/pull/9902/files))

Reply:
Thanks for confirming!